hypershift-operator/controllers/nodepool/token.go

```diff
@@ -101,7 +101,7 @@ func NewToken(ctx context.Context, configGenerator *ConfigGenerator, cpoCapabili
 	// This inconsistency was introduced by https://github.com/openshift/hypershift/pull/3795
 	// See reconcileTokenSecret and https://github.com/openshift/hypershift/pull/4057 for more info on how this is used.
 	// This is kept like this for now to contain the scope of the refactor and avoid backward compatibility issues.
-	hcConfigurationHash, err := supportutil.HashStruct(configGenerator.hostedCluster.Spec.Configuration)
+	hcConfigurationHash, err := supportutil.HashStructWithJSONMapper(configGenerator.hostedCluster.Spec.Configuration, supportutil.NewOmitEmptyFieldJSONMapper("imageStreamImportMode"))
```

Comment:
This needs to be also done in NewMCSParams, otherwise the hash will miss match for any version of the CPO having the most recent API. Agree?